content/tutorials/get-started/get-started-using-php.md

```diff
@@ -0,0 +1,288 @@
+---
+html: get-started-using-php.html
+parent: php.html
+funnel: Build
+doc_type: Tutorials
+category: Get Started
+blurb: Build a PHP app that interacts with the XRP Ledger.
+cta_text: Build an XRP Ledger-connected app
+top_nav_name: PHP
+top_nav_grouping: Get Started
+labels:
+  - Development
+showcase_icon: assets/img/logos/java.svg
+---
+# Get Started Using PHP
+
+This tutorial walks you through the basics of building an XRP Ledger-connected application using [`XRPL_PHP`](https://github.com/AlexanderBuzz/xrpl-php), a PHP library built to interact with the XRP Ledger.
+
+This tutorial is intended for beginners and should take no longer than 30 minutes to complete.
+
+## Learning Goals
+
+In this tutorial, you'll learn:
+
+* The basic building blocks of XRP Ledger-based applications.
+* How to connect to the XRP Ledger using `XRPL_PHP`.
+* How to get an account on the [Testnet](xrp-testnet-faucet.html) using `XRPL_PHP`.
+* How to use the `XRPL_PHP` library to look up information about an account on the XRP Ledger.
+* How to put these steps together to create a simple application.
+
+## Requirements
+
+* `XRPL_PHP` requires PHP 8.1.
+* Recommended for increased performance are the PHP extensions [GMP](http://php.net/manual/en/book.gmp.php) and / or [BCMATH](https://www.php.net/manual/de/book.bc.php)
+
+
+## Installation
+
+`XRPL_PHP` can be installed via [Composer](http://getcomposer.org/doc/00-intro.html):
+
+```console
+composer require hardcastle/xrpl_php
+```
+
+
+## Start Building
+{% set n = cycler(* range(1,99)) %}
```

Comment:
Sadly, this cycler pattern is not possible on Redocly, so we're going to be doing away with it in the near future. It's OK for now (we're going to have to mass-replace the other instances of it) but if you want to be future-proof, you can remove this line and change `{{n.next()}}` instances to actual step numbers.

Reply:
Done

Reply:
Done